### DISCUSSION TITLE
Full-screen visualization

### DISCUSSION THREAD

**Discussion by qdolle15** [neperfepx/neper#759] [Q&A] [answered]
Hello all, 

A simple question on the png output. 
There is a white area surrounding the tessellation with the -V module output in 2D. I would like to have a png with only my tessellation inside in order that the "-imagesize px:py" match with the pixels for the description of the tessellation.
What I have to change/add ? 

Thank you !
Q.

**Comment by rquey** [comment, marked as answer]
Outside of Neper, here is what I can recommend (which does exactly what you want):
```
convert -trim +repage input.png output.png
```

Otherwise, you have to fine-tune the image size ratio and camera angle (which plays horizontally)...
